source/com.microsoft.tfs.client.common.ui/src/com/microsoft/tfs/client/common/ui/teamexplorer/link/TeamExplorerBaseNavigationLink.java

```diff
@@ -27,6 +27,7 @@ public final void clicked(
         final TeamExplorerNavigator navigator,
         final TeamExplorerNavigationItemConfig parentNavigationItem) {
 
+        onClick(shell, context, navigator, parentNavigationItem);
```

Comment:
It's related to Build navigation items (and some other as well). I put it into a separate commit.